src/blocks/modal.css

```diff
@@ -85,8 +73,8 @@ div:hover > ul {
 }
 .modal__close-btn_type_preview {
   background-image: url(../images/Icon-close-btn.svg);
-  top: 0;
-  right: -52px;
+  top: 32px;
+  right: 50x;
```

Comment:
You shouldn't have changed these styles for `modal__close-btn_type_preview`.

The close icon must be **outside** the image modal